pom.xml

```diff
@@ -82,7 +82,7 @@
         <quinoa.version>1.0.5</quinoa.version>
         <surefire-plugin.version>3.0.0-M6</surefire-plugin.version>
         <failsafe-plugin.version>3.0.0-M5</failsafe-plugin.version>
-        <keycloak-admin-client.version>18.0.0</keycloak-admin-client.version>
```

Comment:
Why did you downgrade Keycloak client? I think I've updated this due to some vulnerabilities detected by Snyk. The newer client should be still compatible with older Keycloak.

Reply:
The reason was that the version needs to align to what Quarkus uses, otherwise there is an enforcer rule that fails

Reply:
Okay, in that case we could exclude the dependency brought in Quarkus and use the newer one.

Reply:
That should be fine as well